npm/react/plugins/utils/wrap-devserver.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function returnInjectDevServerFunction (devServer, postPocessConfig = () => {}) {
+  return (...args) => {
+    // Old CT plugin signature: setupDevServer(on, config)
+    if (typeof args[0] === 'function') {
+      const [on, config, additionalOptions] = args
+
+      on('dev-server:start', (options) => {
+        return devServer(options, additionalOptions)
+      })
+
+      return postPocessConfig(config)
```

Comment:
Typo `postPocessConfig` -> `postProcessConfig`

Reply:
fixed in 38f56aa3b71493013e6bb94cb345d48a0c013f26